meta-genivi-dev/recipes-extended/automotive-message-broker/automotive-message-broker_git.bb

```diff
@@ -17,6 +17,7 @@ SRCREV = "c216955d16ca275159891cad296217094d972390"
 
 SRC_URI += "file://amb_allow_sessionbus.patch \
             file://ambd.service \
+            file://ambd_fsa.service \
```

Comment:
Thanks. I see your point but I don't agree. You're launching AMD instance but you're launching it for FSA with FSA-specific configuration (correct?) so that just means FSA has a dependency on this recipe.